redis.c

```diff
@@ -53,6 +53,8 @@ zend_class_entry *redis_exception_ce;
 extern zend_function_entry redis_array_functions[];
 extern zend_function_entry redis_cluster_functions[];
 
+int le_pconnect;
```

Comment:
Maybe a more unique name just to avoid potential conflict?  Maybe `le_redis_pconnect`?

Reply:
Good idea! I'll change it.